### PR TITLE
feat (esl-panel-group): move to separate module

### PR DESCRIPTION
**Pull request by julia-murashko**

### CODE REVIEW

File: src/modules/esl-panel-group/core.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export type {ESLPanelGroupShape} from './core/esl-panel-group.shape';
+
+export * from './core/esl-panel-group';
```
[the remark this answers is not in the document]
Comment (julia-murashko) [reply src/modules/esl-panel-group/core.ts:3]: Yes. it's

File: src/modules/esl-panel-group/core/esl-panel-group.ts [hunk 2]
```diff
@@ -6,9 +6,9 @@ import {memoize} from '../../esl-utils/decorators/memoize';
 import {CSSClassUtils} from '../../esl-utils/dom/class';
 import {ESLMediaRuleList} from '../../esl-media-query/core';
 import {TraversingQuery} from '../../esl-traversing-query/core';
-import {ESLPanel} from './esl-panel';
+import {ESLPanel} from '../../esl-panel/core/esl-panel';
```
[the remark this answers is not in the document]
Comment (julia-murashko) [reply src/modules/esl-panel-group/core/esl-panel-group.ts]: Fixed. Thank you!

File: src/modules/esl-panel-group/core/esl-panel-group.ts [hunk 3]
```diff
@@ -6,9 +6,9 @@ import {memoize} from '../../esl-utils/decorators/memoize';
...
 
-import type {PanelActionParams} from './esl-panel';
+import type {PanelActionParams} from '../../esl-panel/core/esl-panel';
```
[the remark this answers is not in the document]
Comment (julia-murashko) [reply src/modules/esl-panel-group/core/esl-panel-group.ts]: Fixed. Thank you!

File: src/modules/esl-panel/core/esl-panel.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ import {bind} from '../../esl-utils/decorators/bind';
 import {afterNextRender} from '../../esl-utils/async/raf';
 import {attr, jsonAttr} from '../../esl-base-element/core';
 import {ESLToggleable} from '../../esl-toggleable/core';
-import {ESLPanelGroup} from './esl-panel-group';
+import {ESLPanelGroup} from '../../esl-panel-group/core/esl-panel-group';
```
[the remark this answers is not in the document]
Comment (julia-murashko) [reply src/modules/esl-panel/core/esl-panel.ts]: Fixed. Thank you!

**Review by ala-n — APPROVED** [review]
❤️ 👍🏻

**Review by yadamskaya — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Pull request merged**